sortby.rb

```diff
@@ -0,0 +1,38 @@
+def bubble_sort_items(array)
+  items_unsorted = true
+  items_unsorted, array = sorting(array) while items_unsorted
+  array
+end
+
+def sorting(array)
+  sorted = true
+  (0...(array.count - 1)).each do |index|
+    if array[index] > array[index + 1]
+      sorted = false
+      array[index], array[index + 1] = [array[index + 1], array[index]]
+    end
+  end
+  [!sorted, array]
+end
```

Comment:
This should be your main method and it should integrate the one on the top. Please make the necessary adjustments.